client/pdo/client/scripts/ShellCLI.py

```diff
@@ -154,12 +157,18 @@ def Main() :
     if options.key_dir :
         config['Key']['SearchPath'] = options.key_dir
 
-    # set up the service configuration
+   # set up the service configuration
     if config.get('Service') is None :
         config['Service'] = {
+            'EnclaveServiceNames' : [],
             'EnclaveServiceURLs' : [],
-            'ProvisioningServiceURLs' : []
+            'ProvisioningServiceURLs' : [],
+            'EnclaveServiceDatabaseFile' : None
```

Comment:
seems like you should for all of these clients, simply load a database & then take a set of parameters that add, remove, or get things from the db. the implication again is that the db is created once for the client.